StRoot/StSpinPool/StFcsTriggerSimMaker/StFcsTriggerSimMaker.h

```diff
@@ -37,9 +37,10 @@ class StFcsTriggerSimMaker : public StMaker{
     void setWriteQaTree(char* filename) {mQaTreeFilename=filename;};
     void setWriteQaHist(char* filename) {mQaHistFilename=filename;};
     void setReadPresMask(char* filename) {mPresMask=filename;};
-    void setSimMode(int v, int tb=52) {mSimMode=v; mTrgTimebin=tb;}
+    void setSimMode(int v, int tb=50) {mSimMode=v; mTrgTimebin=tb;}
```

Comment:
Sim mode... trg time bin... seems like something that should be in a database.  But again, pool directory, so the presumption is this is not production code...